Gemfile

```diff
@@ -2,5 +2,7 @@ source "https://www.rubygems.org"
 
 group :development do
   # Needed by Travis so we can run `bundle exec rake`
+  # Do not install rake, because it requires Ruby version >= 1.9
```

Comment:
Thanks for the comment.
Firstly I had wanted to run 'bundle exec rake' on Travis.
And I added  ```gem 'rake'``` on Gemfile. Then I found it did not work on Travis.
So, I will remove this line and not submit in my PR.
But I think to write some comments for this line are useful to prevent someone to add same logic ```gem 'rake'```  like me in the future.